apps/teams-test-app/src/components/AppAPIs.tsx

```diff
@@ -7,9 +7,18 @@ const GetContext = (): ReactElement =>
   ApiWithoutInput({
     name: 'getContextV2',
     title: 'Get Context',
-    onClick: async () => {
-      const context = await app.getContext();
-      return JSON.stringify(context);
+    onClick: {
+      withPromise: async () => {
+        const context = await app.getContext();
+        return JSON.stringify(context);
+      },
+      withCallback: setResult => {
+        const callback = (context: Context): void => {
+          setResult(JSON.stringify(context));
+        };
+        const context = getContext(callback);
+        return JSON.stringify(context);
```

Comment:
In the callback version `getContext` doesn't really return anything, does it?

```
export function getContext(callback: (context: Context) => void): void
```

so I'm not exactly sure what you're printing out out here. If I remember correctly before we moved to promises we used to initially write out `return 'API_NAME_HERE' + noHostSdkMsg;`, so in this case I think you should return `return 'getContext()' + noHostSdkMsg;` and let the callback update it to the actual values when invoked.

Reply:
Sorry I didn't know that, I will make that change now